Add API use case dropdowns (#33438)
DataDog/documentation: content/en/agent/fleet_automation/remote_management.md

[18]: https://app.datadoghq.com/fleet
{{% /collapse-content %}}

{{% collapse-content title="Configure Agents via API" level="h4" expanded=false id="id-for-anchoring" %}}

> Fleet Automation provide an API to apply configuration updates to your Agents programmatically. You can deploy configuration changes to any group of hosts using filter queries, supplying either full configuration files or targeted patches. Fleet Automation does not support all Agent configuration fields, including settings related to Agent connection or secrets such as site, API keys, and other authentication parameters. Configuration can be pushed on demand or integrated into your existing automation workflows. For full details, see the [Fleet Automation API][19].

Reviewer: ```suggestion
Fleet Automation provides an API to apply configuration updates to your Agents programmatically. Deploy changes to any group of hosts using filter queries, supplying either full configuration files or targeted patches. Fleet Automation does not support all Agent configuration fields, and settings related to Agent connection or secrets (site, API keys, and other authentication parameters) cannot be managed through the API. Push configuration on demand or integrate it into your existing automation workflows. For full details, see the [Fleet Automation API][19].
```

[16]: https://app.datadoghq.com/fleet/agent-management
[17]: https://docs.datadoghq.com/agent/remote_config/?tab=configurationyamlfile#configuration-order-precedence
[18]: /agent/fleet_automation#view-agent-audit-trail-events

> [19]: https://docs.datadoghq.com/api/latest/fleet-automation/

Reviewer: ```suggestion
[19]: /api/latest/fleet-automation/
```
